client/dashboard/sites/deployment-list/index.tsx

```diff
@@ -68,9 +29,11 @@ function SiteDeployments() {
 					title={ __( 'Deployments' ) }
 					actions={
 						<>
-							<Button variant="secondary" __next40pxDefaultSize>
-								{ __( 'Manage repositories' ) }
-							</Button>
+							<Link to="/sites/$siteSlug/settings/repositories" params={ { siteSlug } }>
```

Comment:
We can use `RouterLinkButton` 😄 

https://github.com/Automattic/wp-calypso/blob/15c917121cecabe7b05d99ccff476a7a1d4b6153/client/dashboard/components/router-link-button/index.tsx#L6